openstack/keppel/values.yaml

```
@@ -42,6 +42,10 @@ keppel:
   # that all Keppels accept HTTP requests for this domain name)
   anycast_domain_name: ""
 
+  anycast_nginx_tag: "1.28"
```

We currently only have
```suggestion
  anycast_nginx_tag: "mainline-slim-latest"
```
in that repo.